src/index.html

```diff
@@ -11,17 +11,410 @@
+              <div class="stars__star stars__star--color"></div>
+              <div class="stars__star stars__star--color"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__quantity-reviews">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <span class="card__descr">Price:</span>
+            <span class="card__value">$2,199</span>
+          </div>
+
+          <a
+            href="#"
+            class="card__btn"
+          >
+            buy
+          </a>
+        </div>
+      </div>
+      <div class="card">
+        <div class="card__image">
+          <img
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+          />
+        </div>
+
+        <div class="card__content">
+          <div class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__reviews">
+            <div class="stars card__stars">
+              <div class="stars__star stars__star--color"></div>
+              <div class="stars__star stars__star--color"></div>
+              <div class="stars__star stars__star--color"></div>
+              <div class="stars__star stars__star--color"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__quantity-reviews">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <span class="card__descr">Price:</span>
+            <span class="card__value">$2,199</span>
+          </div>
+
+          <a
+            href="#"
+            class="card__btn"
+          >
+            buy
+          </a>
+        </div>
+      </div>
+      <div class="card">
+        <div class="card__image">
+          <img
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+          />
+        </div>
+
+        <div class="card__content">
+          <div class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__reviews">
+            <div class="stars card__stars">
+              <div class="stars__star stars__star--color"></div>
+              <div class="stars__star stars__star--color"></div>
+              <div class="stars__star stars__star--color"></div>
+              <div class="stars__star stars__star--color"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__quantity-reviews">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <span class="card__descr">Price:</span>
+            <span class="card__value">$2,199</span>
+          </div>
+
+          <a
+            href="#"
+            class="card__btn"
+          >
+            buy
+          </a>
+        </div>
+      </div>
```

Comment:
It seems like you are repeating the same block of code for each card. To make your code DRY (Don't Repeat Yourself), consider creating a reusable component for the card and using it for each product. This will make your code more maintainable and easier to read.